src/main/java/htsjdk/variant/utils/SAMSequenceDictionaryExtractor.java

```diff
@@ -111,7 +144,10 @@ SAMSequenceDictionary extractDictionary(final File intervalList) {
             applicableExtensions = extensions;
         }
 
+        /** @deprecated in favor of extractDictionary(final Path file) */
+        @Deprecated
         abstract SAMSequenceDictionary extractDictionary(final File file);
```

Comment:
Wouldn't it be simpler to make this non-abstract and implement it as
```
return extractDictionary(file.toPath());
```
?

Reply:
done.